modules/sdk-core/src/bitgo/wallet/wallets.ts

```diff
@@ -538,6 +540,65 @@ export class Wallets implements IWallets {
       .result();
   }
 
+  /**
+   * Re-share wallet with existing spenders of the wallet
+   * @param walletId
+   * @param userPassword
+   */
+  async reshareOfcAccountWithSpenders(walletId: string, userPassword: string): Promise<void> {
```

Comment:
```suggestion
  async reshareWalletWithSpenders(walletId: string, walletPassword: string): Promise<void> {
```

Reply:
This is the userPassword and not walletPassword.

<img width="1728" alt="Screenshot 2024-04-18 at 12 36 19 PM" src="https://github.com/BitGo/BitGoJS/assets/131337791/74af269a-64fd-47b5-bd39-6586f0c766ba">